tests/python/driver/tvmc/test_frontends.py

```diff
@@ -0,0 +1,196 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+import os
+import tarfile
+
+import pytest
+
+from tvm.ir.module import IRModule
+
+from tvm.driver import tvmc
+from tvm.driver.tvmc.common import TVMCException
+
+
+def test_get_frontend_names_is_list():
+    sut = tvmc.frontends.get_frontend_names()
+    assert type(sut) is list
+
+
+def test_get_frontends_contains_only_strings():
+    sut = tvmc.frontends.get_frontend_names()
+    assert all([type(x) is str for x in sut]) is True
+
+
+def test_get_frontend_by_name_valid():
+    # some CI environments wont offer TensorFlow/Keras, so skip in case it is not present
+    pytest.importorskip('tensorflow')
+
+    sut = tvmc.frontends.get_frontend_by_name("keras")
+    assert type(sut) is tvmc.frontends.KerasFrontend
+
+
+def test_get_frontend_by_name_invalid():
+    with pytest.raises(TVMCException) as e:
+        def f():
+            tvmc.frontends.get_frontend_by_name("unsupported_thing")
+        f()
+    assert 'unrecognized frontend' in str(e.value)
```

Comment:
Should be able to simplify to the following. If it raises an exception than this test is passed so we don't care what it printed; otherwise we will see a pytest error of not raising an expected exception.

```suggestion
        tvmc.frontends.get_frontend_by_name("unsupported_thing")
```